src/core/github.js

```diff
@@ -9,6 +9,44 @@ import { pub } from "core/pubsubhub";
 
 export const name = "core/github";
 
+var body = document.body;
+var head = document.head;
+var issueBase;
+
+var fileIssueLink = document.createElement('button');
+fileIssueLink.setAttribute('data-balloon','file an issue related to the text selected');
+fileIssueLink.setAttribute('data-balloon-pos','left');
+fileIssueLink.classList.add('removeOnSave');
+fileIssueLink.id = 'respec-file-issue-btn';
+fileIssueLink.innerHTML = 'File Issue';
+fileIssueLink.onclick = function () {
+  window.location.href = fileIssueLink.getAttribute('data-href');
+  fileIssueLink.style.display = 'none';
+};
+
+var link = document.createElement('link');
+link.rel = 'stylesheet';
```

Comment:
Note: this link also needs `removeOnSave`, as we don't want ballon.css in the final spec.

Reply:
Cool, will do this too